tools/plugins/network/src/main/kotlin/net/corda/cli/plugins/network/MemberList.kt

```diff
@@ -0,0 +1,90 @@
+package net.corda.cli.plugins.network
+
+import net.corda.cli.plugins.common.RestClientUtils.createRestClient
+import net.corda.cli.plugins.common.RestCommand
+import net.corda.membership.rest.v1.MemberLookupRestResource
+import net.corda.membership.rest.v1.types.response.RestMemberInfo
+import picocli.CommandLine
+
+@CommandLine.Command(name = "members", description = ["List members"])
+class MemberList : RestCommand(), Runnable {
+    override fun run() {
```

Comment:
After the change in my previous comment, this can simply be -
```suggestion
    override fun run() {
        getMembersList()
    }
```